.drone.yml

```diff
@@ -135,7 +143,9 @@ steps:
       restore: true
 
   - name: ui-tests
-    image: codedotorg/code-dot-org:1.5.4
+    # see: docker/dockerfiles/Dockerfile
+    image: codedotorg/code-dot-org:1.6.0
+    pull: always
```

Comment:
This makes `ui-tests` behavior consistent with `unit-tests` (which already is set to pull: always), so they're both running on the same version of the `codedotorg/code-dot-org` image if/when there are multiple pushes to the same tag.